tests/www/test_views.py

```diff
@@ -474,6 +474,11 @@ def test_tree(self):
         resp = self.client.get(url, follow_redirects=True)
         self.check_content_in_response('runme_1', resp)
 
+    def test_tree_subdag(self):
+        url = 'tree?dag_id=example_subdag_operator.section-1'
+        resp = self.client.get(url, follow_redirects=True)
+        self.check_content_in_response('section-1', resp)
```

Comment:
This message would appear even even if the dag couldn't be found (from the "Dag X couldn't be found" message) - could we check for something else in the content?

Reply:
Sure, switched to the name of one of the tasks inside the subDAG